pipeline/schema.py

```diff
@@ -291,7 +291,7 @@ def validate_schema(schema, message, logger=None):
 offer = SchemaAllRequired({
     'commercialModelType': CommercialModelType,
     'licensee': str,
-    Optional('price'): str,
+    Optional('price'): Any(str, None),
     'territoryCode': str,
     'useType': UseType,
     'validFrom': Any(None, OffsetAwareDatetime),
```

Comment:
See [this](https://github.com/iheartradio/pipeline/pull/80#issuecomment-402813976) comment.